packages/fxa-payments-server/src/routes/Product/SubscriptionUpgrade/en.ftl

```diff
@@ -10,3 +10,11 @@ sub-change-submit = Confirm change
 sub-update-current-plan-label = Current plan
 sub-update-new-plan-label = New plan
 sub-update-total-label = New total
+sub-update-one-time-charges = One time charges (due today)
```

Comment:
Why plural "charges"? If it's one-time, wouldn't that make it a single charge?

Also, should it be [one-time](https://grammarist.com/spelling/one-time-onetime/) instead of _one time_?

Reply:
"charges" was in the mock-up, but you're right, it looks like a typo. And good catch. Should be "One-time". My bad. Thank you. Both have been corrected!